alf/algorithms/algorithm.py

```diff
@@ -737,17 +737,41 @@ def update_with_gradient(self, loss_info, valid_masks=None, weight=1.0):
         return loss_info, all_params
 
     def after_update(self, training_info):
-        """Do things after complete one gradient update (i.e. ``update_with_gradient()``).
+        """Do things after completing one gradient update (i.e. ``update_with_gradient()``).
         This function can be used for post-processings following one minibatch
         update, such as copy a training model to a target model in SAC, DQN, etc.
 
         Args:
-            training_info (nested Tensor): information collected for training.
+            training_info (TrainingInfo): information collected for training.
                 It is batched from each ``info`` returned by ``rollout_step()`` or
                 ``train_step()``.
         """
         pass
 
+    def after_train_iter(self, training_info=None):
```

Comment:
Can you also update the comments for RLAlgorithm about all the major hooks?

Reply:
updated